hack/deploy-hub-local/build-edgecluster.sh

```diff
@@ -142,20 +158,23 @@ EOF
   if [ "${HUB_ARCHITECTURE}" = "compact" ]; then
     # Add the single worker
     worker=0
-    WORKERUID=$(kcli info vm edgecluster${edgecluster}-cluster-w${worker} -f id -v)
+    WORKERUID=$(kcli info vm ${CLUSTER_NAME}${edgecluster}-cluster-w${worker} -f id -v)
+    WORKER_NETS=$(kcli info vm -oyaml -fnets ${CLUSTER_NAME}${edgecluster}-cluster-w0)
+    BARE_NET_MAC=$(echo "${WORKER_NETS}" | yq e -j | jq -r ".nets[0].mac")
+    ZTPFW_MAC=$(echo "${WORKER_NETS}" | yq e -j | jq -r ".nets[0].mac")
```

Comment:
```suggestion
    ZTPFW_MAC=$(echo "${WORKER_NETS}" | yq e  ".nets[0].mac" - )
```

Reply:
I tried this a couple of times and it just kept failing, hence the `-j` hack. FWIW, Karim just released a json output for `kcli` so we can switch to that as soon as we update the servers.